db/schema.rb

```diff
@@ -99,6 +99,19 @@
     t.index ["version_id"], name: "index_dependencies_on_version_id"
   end
 
+  create_table "events_user_events", force: :cascade do |t|
```

Comment:
:thinking: seems there are leftovers from following PRs

Reply:
I recommend [hookup](https://github.com/tpope/hookup) to help prevent this 👍🏻